src/web/src/pages/Main.vue

```diff
@@ -310,9 +310,10 @@ export default {
             exclusive_date_end.setDate(course.date_end.getDate() + 1);
             semester = session.semester;
             calendarBuilder.addEvent(
-              `Class: ${course.title}`,
+              `Class: ${course.full_title || course.title}`,
               'LEC day',
-              session.location,
+//              session.location,
```

Comment:
If you're going to leave location present but commented, you should also have professor there was as well

Reply:
location is its own special field in the ICS stuff. I can add a comment to indicate professor should be added to the description